test_configuration_item.py

```diff
@@ -0,0 +1,29 @@
+from main import ConfigurationItem, ProjectManConfigTypeInvalidError
+import pytest
+
+items = ["issues", "prs", "all"]
+
+get_conf_item = ConfigurationItem(
+    item_type="issues",
+    labels=["bug"],
+    assignees=["thepetk"],
+    milestones=["12/12/2023"],
+    last_updated_on="fefde",
+    created_on="hfgbdv",
+    closed_on="rfcrsd",
+    reviewers=None,
+)
+
+
+def test_get_configuration_item_success():
+    for item_type in items:
+        if item_type == "issues":
```

Comment:
This is correct. Although we can run the test for all the `items` not only for `item_type == "issues"`